speedtest-tracker/docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  speedtest-tracker:
+    image: ${SPEEDTEST_TRACKER_IMAGE}
+    restart: unless-stopped
+    #ports:
+    #    - 8080:80
```

Comment:
these old example ports suggest that it was designed to listen on both ports, so is there a config to turn off the second port?